packages/allure-codeceptjs/test/retries.test.ts

```diff
@@ -0,0 +1,47 @@
+const allurePlugin = require( "../src/index.ts");
+describe('handleRetries', () => {
```

Comment:
i guess i need simplify this test later

Reply:
@vovsemenv could you please run build again ?)

Reply:
looks fine for now but i imagine we can do it like this
```ts
import { expect } from "@jest/globals";
import { runTests } from "./utils/run-tests";

test("retries", async () => {
  const res = await runTests({
    files: {
      "retries.test.js": /* js */ `
        Feature("retries-feature");
        Scenario("Failing feature", ({ I }) => {
          //@ts-ignore
          if (!global.shouldPass) {
            throw new Error("Test Fail");
          }
          //@ts-ignore
          global.shouldPass = true;
        }).retry(1);
      `,
    },
  });
  expect(res.tests.length).toBe(2);
  expect(res.tests).toEqual(
    expect.arrayContaining([
      expect.objectContaining({
        fullName: "retries.test.js#Failing feature",
      }),
    ]),
  );
});

```